tests/test_example.py

```diff
@@ -0,0 +1,12 @@
+from open_parser import OpenParser
+
+if __name__ == "__main__":
+
+    example_apikey = "1PW5Li5aif4wuv9bG5f1KaEJkJEnxrmF95esi4FM"
```

Comment:
lolz. you should remove api key before publish your PR for review and I am going to delete this key, so you can create a new one.